packages/altive_lints/lib/src/lints/avoid_hardcoded_color.dart

```diff
@@ -45,11 +45,10 @@ class AvoidHardcodedColor extends DartLintRule {
     CustomLintContext context,
   ) {
     context.registry.addInstanceCreationExpression((node) {
-      final typeName =
-          node.staticType?.getDisplayString(withNullability: false);
+      final typeName = node.staticType?.getDisplayString();
 
       if (typeName == 'Color') {
-        reporter.reportErrorForNode(code, node);
+        reporter.atNode(node, _code);
```

Comment:
After the update, LintRule still appears to have `final LintCode code`, but for some reason `code` is not available in `dynamic`.
Fixed to use `_code`.